### PR TITLE
user repo use defer seq

### PR DESCRIPTION
**Pull request by yingjieMiao**
@raymondng @ahsu1230

### CODE REVIEW

**Comment by raymondng** [comment]
:+1:

File: kifi-backend/modules/common/conf/evolutions/shoebox/114.sql [hunk 1]
```diff
@@ -9,7 +9,7 @@
 CREATE SEQUENCE user_sequence;
 ---
 
-ALTER TABLE user add seq Int NOT NULL DEFAULT 0;
+ALTER TABLE user add seq bigint(20) NOT NULL DEFAULT 0;
```
Comment (ahsu1230) [inline kifi-backend/modules/common/conf/evolutions/shoebox/114.sql:12]: nice catch :+1:

File: kifi-backend/modules/common/conf/evolutions/shoebox/114.sql [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
 CREATE INDEX user_i_seq ON user(seq);
 
 INSERT INTO evolutions (name, description) VALUES ('114.sql', 'add sequence number to user');
```
Comment (ahsu1230) [inline kifi-backend/modules/common/conf/evolutions/shoebox/114.sql:15]: maybe a newline after this line?

Comment (yingjieMiao) [inline kifi-backend/modules/common/conf/evolutions/shoebox/114.sql:15]: oh, good catch. It was an old 'bug'.

**Pull request merged**